Configuration/Geometry/python/GeometryDD4hepRecoDB_cff.py

```diff
@@ -0,0 +1,60 @@
+import FWCore.ParameterSet.Config as cms
+
+# This config was generated automatically using generate2021Geometry.py
+# If you notice a mistake, please update the generating script, not just this config
+
+from Geometry.EcalCommonData.ecalSimulationParameters_cff import *
+from Geometry.HcalCommonData.hcalDDDSimConstants_cff import *
+from Geometry.MuonNumbering.muonGeometryConstants_cff import *
+from Geometry.MuonNumbering.muonOffsetESProducer_cff import *
```

Comment:
If I read this file correctly, the contents is the same as for DDD file except it adds
```py
from Geometry.EcalCommonData.ecalSimulationParameters_cff import *
from Geometry.MuonNumbering.muonOffsetESProducer_cff import *
from Geometry.TrackerGeometryBuilder.trackerParameters_cff import *
trackerGeometryDB.applyAlignment = cms.bool(False)
from Geometry.CaloEventSetup.CaloGeometryBuilder_cfi import *
CaloGeometryBuilder = cms.ESProducer("CaloGeometryBuilder",
    SelectedCalos = cms.vstring("HCAL",
                                "ZDC",
                                "EcalBarrel",
                                "EcalEndcap",
                                "EcalPreshower",
                                "TOWER",
    )
)
```
and removes the `run3_GEM` customization.

With this level of re-use I think it would be clearer to use a single `GeometryRecoDB_cff.py` file and add the necessary new imports/loads there for `dd4hep` Modifier.